applications/vanilla/controllers/class.discussionscontroller.php

```diff
@@ -841,4 +850,15 @@ public function tagged() {
         $this->View = c('Vanilla.Discussions.Layout') == 'table' && $this->SyndicationMethod == SYNDICATION_NONE ? 'table' : 'index';
         $this->render($this->View, 'discussions', 'vanilla');
     }
+
+    /**
+     * Allows pages to opt-in to the category following filter.
+     *
+     * @param string $url
+     */
+    public function setCategoryFollowingList(string $url = null) {
```

Comment:
This name is kinda misleading. The word "set" makes me think I'm overwriting whatever value might currently be there.

Reply:
Why'd you make `$url` optional? It seems like you'd want to make that required, since all this function does is take the value you give it and load it into the array.